base/client.jl

```diff
@@ -180,8 +180,8 @@ function eval_user_input(@nospecialize(ast), show_value::Bool)
                 errcount, lasterr = 0, ()
             else
                 ast = Meta.lower(Main, ast)
-                value = eval(Main, ast)
-                eval(Main, Expr(:body, Expr(:(=), :ans, QuoteNode(value)), Expr(:return, nothing)))
+                value = Main.eval(ast)
+                Main.eval(Expr(:body, Expr(:(=), :ans, QuoteNode(value)), Expr(:return, nothing)))
```

Comment:
This part of the changes seems strictly wrong. `Main.eval` is a completely random function that's entirely unrelated to the `Core.eval(Main, "")` function this previously called into.

Reply:
Well, it's the `eval` function that I put there, but yes I suppose somebody could overwrite it.